MVCCore.Models/Login.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+using System.Diagnostics.CodeAnalysis;
+
+
+namespace MVCCore.Models
+{
+    public class Login  
+    {
```

Comment:
Remove data validation attribute when you using fluent validation, this fires before the fluent in the .net pipeline

Reply:
Removed. Now fluent validation is working